src/index.ts

```diff
@@ -134,7 +137,8 @@ export default function onionify<So, Si>(
       .drop(1);
     sources[name] = new StateSource<any>(state$, name) as any;
     const sinks = main(sources as So);
-    reducerMimic$.imitate(sinks[name]);
+    const imitation$ = sinks[name] && xs.fromObservable(sinks[name]);
```

Comment:
Because without it tests failing, there is two tests with undefined sink and when we do xs.fromObservable(undefined) it fails